### PR TITLE
Add IIT_V6 to support 6-element vectors in intrinsics.

### PR DESCRIPTION
**Pull request by rampitec**
Needed for the future patch.

### CODE REVIEW

File: llvm/include/llvm/IR/Intrinsics.td
Comment (chapuni) [inline llvm/include/llvm/IR/Intrinsics.td:275]: You should avoid lower numbering unless it is used often. `<=16` is for short encoders.

**Review by chapuni — CHANGES_REQUESTED** [review]

File: llvm/include/llvm/IR/Intrinsics.td
Comment (rampitec) [reply llvm/include/llvm/IR/Intrinsics.td]: Thanks! Moved to the end.

**Review by chapuni — APPROVED** [review]
LGTM. Please wait for one more approval.

**Comment by rampitec** [comment]
> LGTM. Please wait for one more approval.

Ack. Thanks for review!

**Review by arsenm — APPROVED** [review]

**Pull request merged**
